src/index.ts

```diff
@@ -273,24 +261,22 @@ export class EIDRConnector extends BaseConnector {
       }
     }
 
-    throw new EIDRError(
-      'Unrecognized response',
-      500,
-      'Unrecognized response from registry',
-    )
+    throw new EIDRError('Unrecognized response', 500,
+      'Unrecognized response from registry')
   }
 
   public async resolve(id: string, type = 'Full') {
     if (
+      type !== 'AlternateIDs' &&
+      type !== 'DOIKernel' &&
       type !== 'Full' &&
+      type !== 'LinkedAlternateIDs' &&
+      type !== 'Provenance' &&
       type !== 'SelfDefined' &&
-      type !== 'LinkedAlternateIDs'
+      type !== 'Simple'
```

Comment:
Consider using
`if (!['AlternateIDs','DOIKernel','Full','LinkedAlternateIDs','Provenance','SelfDefined','Simple'].includes(type))`

Reply:
I though about that, but the "&&"s seemed more readable